### ISSUE TITLE
"*.whl is not a supported wheel on this platform" when building package with rc.* versions

### ISSUE THREAD

**Issue by MrMrRobat** [dephell/dephell#192]
Getting this error while installing packages with `rc.*` versions built with `dephell project build`
```bash
ERROR: PackageName-0.1.0-rc.1-py3-none-any.whl is not a supported wheel on this platform.
```
Building with package `python3 setup.py sdist bdist_wheel` or changing "`0.1.0-rc.1-py3`" to "`0.1.0rc1-py3`" in package name fixes the issue.

**Upd.**
However there is no workaround for `dephell project test` — with this bug it's entirely broken for packages with `rc.*` versions

**Comment by orsinium** [comment]
DepHell uses pip inside of `install` command. This is an error inside of pip, I think. I guess, this is because of weak parsing of wheel names by `-` separator

**Comment by MrMrRobat** [comment]
But on the other hand, `dephell project build` generates names different from `python3 setup.py sdist bdist_wheel`. We can't change how pip parses packages (at least not so soon or so easy), but can generate proper names for it.

**Comment by orsinium** [comment]
Right. What name is generated by both methods?

**Comment by MrMrRobat** [comment]
```bash
dephell project build       ->        PackageName-0.1.0-rc.1-py3-none-any.whl
python3 setup.py sdist bdist_wheel -> PackageName-0.1.0rc1-py3-none-any.whl
```

**Comment by MrMrRobat** [comment]
Seems to be fixed in #203, thanks!
